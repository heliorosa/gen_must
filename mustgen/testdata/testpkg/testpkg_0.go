package testpkg

func doThing() (int, error) {
	//@gen_must
	return 0, nil
}
