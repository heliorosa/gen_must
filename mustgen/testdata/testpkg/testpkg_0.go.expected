// Code generated - DO NOT EDIT.
// This file is auto generated by gen_must and any manual changes will be lost.

package testpkg

// mustDoThing has the behavior of doThing, except it panics on error
func mustDoThing() int {
	var0, err := doThing()
	if err != nil {
		panic(err)
	}
	return var0
}
